servers/zms/src/main/java/com/yahoo/athenz/zms/store/impl/jdbc/JDBCConnection.java

```diff
@@ -538,6 +543,8 @@ public JDBCConnection(Connection con, boolean autoCommit) throws SQLException {
         con.setAutoCommit(autoCommit);
         transactionCompleted = autoCommit;
         objectMap = new HashMap<>();
+        roleTagsLimit = Integer.getInteger(ZMSConsts.ZMS_PROP_QUOTA_ROLE_TAG, 25);
```

Comment:
we shouldn't keep fetching the system property for every new connection. Instead these values should be fetched as part of the JDBCObjectStore should fetch them as part of its constructor and then set the when a new connection is returned - something similar to the setOperationTimeout method the jdbc connection has. This way we only fetch the settings once and not over and over again.

Reply:
sure. originally it was static const.. forgot that this is a heavy created object..